docs/misc/kafka-pv-gce.yaml

```diff
@@ -12,5 +12,4 @@ spec:
   persistentVolumeReclaimPolicy: Retain
   gcePersistentDisk:
     pdName: kubeless-kafka
-    fsType: ext4
-
+    fsType: ext4
```

Comment:
please restore the empty line at the end

Reply:
ok,i will do that